label_studio/server.py

```diff
@@ -177,6 +177,11 @@ def _create_user(input_args, config):
     user.active_organization = org
     user.save(update_fields=['active_organization'])
 
+    # Enable legacy API tokens if the flag or environment variable is set
```

Comment:
Could you instead update this in [organizations.functions.create_organization](https://github.com/HumanSignal/label-studio/blob/050837520e0a33baaf6af47c30b08b14266358cf/label_studio/organizations/functions.py#L7)? That's where we set the token settings to begin with, and I'd like to keep that logic as consolidated as possible 🙂

Reply:
Sure. I will deal with it when I'm next it in front of the computer

Reply:
@pakelley Done. Please feel free to make any edits you like if you want to change anything